### ISSUE TITLE
using it with foreign language

### ISSUE THREAD

**Issue by samreenkazi** [jind11/word2vec-on-wikipedia#3]
Hello

am done with corpus preparation, but how do i run this google utility step u mentioned with Urdu language corpus
<
 ./ word2vec -train ../processed/wiki -output ../results/enwiki.skip.size300.win10.neg15.sample1e-5.min10.bin -cbow 0 -size 300 -window 10 -negative 15 -hs 0 -sample 1e-5 -threads 24 -binary 1 -min-count 10
 >

another thing,this text9.zip file should be used only when we have trained on english word2vec, what should i replace it with incase of urdu language

**Comment by jind11** [comment]
hi, for other languages than English, you just need to replace the train file. in the example, train file is set by the command "--train ../processed/wiki", here the wiki file can be in any language, if it is English, then it would produce English embeddings.

**Comment by samreenkazi** [comment]
Hi,
Thanks for reply
Whenever am trying to execute last command in ur readme, its producing error cant recognize word2vec command , my processed corpus is also in processed/wiki .

**Comment by jind11** [comment]
have you downloaded the word2vec toolkit?

**Comment by samreenkazi** [comment]
Yes i have downloded word2vec toolkit as well, are there further dependencies which need to be installed for running googleword2vec, and am i suppose to replace text8.zip as well from googleword2vec folder with anything related to urdu , or this .zip folder is not used at all if i am already providing training data set in processed/wiki.
